src/dhclientv4_wrapper.c

```diff
@@ -253,7 +253,7 @@ static void process_dhclient_script_callback(const int argc, char *argv[])
         /* If the user tells dhclient to call this program as the script
        (-isf script option), format and print the dhclient result nicely. */
 
-    debug("%s,%s,%s,%s,%s,%s,%s,%s\n",
+    fprintf(stderr, "%s,%s,%s,%s,%s,%s,%s,%s\n",
```

Comment:
Do we want this as stderr?  or should it be pulled back out too?

Reply:
yes we need it to stderr for some reason... I disliked the former debug( under #if DEBUG flag which may lead to some misunderstanding i.e. why after disabling debug prints it stops working.

Reply:
Are we still printing to stderr instead of stdout for normal output then?